docs/0.3/references/cli/grid-agent-create.1.md

```diff
@@ -1,27 +1,104 @@
-% GRID-AGENT-CREATE(1) Cargill, Incorporated | Grid Commands
+% GRID-AGENT-CREATE(1) Cargill, Incorporated | Grid
+
 <!--
-  Copyright 2018-2020 Cargill Incorporated
```

Comment:
I believe for existing copyright headers, the date range should just be updated rather than removing the range and putting the current year

Reply:
That man page in main only has `2021`: https://github.com/hyperledger/grid/blob/main/cli/man/grid-agent-create.1.md.. I believe this man page was circa 0.1